x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -2819,13 +2819,6 @@
     "tsvb.annotationsEditor.rowTemplateLabel": "行模板（必需）",
     "tsvb.annotationsEditor.timeFieldLabel": "时间字段（必需）",
     "tsvb.axisLabelOptions.axisLabel": "每 {unitValue} {unitString}",
-    "tsvb.axisLabelOptions.daysLabel": "天",
-    "tsvb.axisLabelOptions.hoursLabel": "小时",
-    "tsvb.axisLabelOptions.minutesLabel": "分钟",
-    "tsvb.axisLabelOptions.monthsLabel": "个月",
-    "tsvb.axisLabelOptions.secondsLabel": "秒",
-    "tsvb.axisLabelOptions.weeksLabel": "周",
-    "tsvb.axisLabelOptions.yearsLabel": "年",
```

Comment:
I think we don't have to remove these: don't are you using these for the translation of the unitString, but just with a different id like `tsvb.getInterval.hoursLabel`?